app/main.py

```diff
@@ -1,8 +1,31 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
```

Comment:
The `people` dictionary is a class variable, meaning it is shared across all instances of the `Person` class. This can lead to unexpected behavior if multiple `Person` objects are created, as they will all share the same `people` dictionary. Consider using an instance variable instead.